modules/ROOT/pages/migration/upgrading-ocis.adoc

```diff
@@ -21,6 +21,11 @@ IMPORTANT: When upgrading from an older release to the desired one, *ALL* upgrad
 
 * The space index of the decomposedFS will be automatically migrated from symlinks to messagepack. This migration needs preparation to run successfully.
 * The xref:deployment/container/orchestration/orchestration.adoc#using-our-helm-charts-with-infinite-scale[Helm Chart] has been upgraded.
+* Environment variables marked for deprecation have been removed
+
+=== Breaking Changes Requiring Manual Intervention
+
+* Client pool selectors have changed from IP addresses `127.0.0.1:9xxx` to service names `com.owncloud.api.*`
```

Comment:
shouldn't we mention the defaults we use? see: https://github.com/fschade/ocis/blob/0ecb240083d35c2597714c74c740fabdcfea4576/services/gateway/pkg/config/defaults/defaultconfig.go#L55-L64

Reply:
see comment below